src/components/Chart/Chart.tsx

```diff
@@ -2,12 +2,19 @@ import React from "react";
 import "./Chart.css";
 import ChartBar from "./ChartBar";
 
-const Chart = (props) => {
-  const numValues = props.dataValue.map((expense) => expense.value);
+type labelFields = {
+  label: string;
+  value: number;
+};
+
+const Chart = (props: { dataValue: labelFields[] }) => {
```

Comment:
Setting the props type here isn't incorrect. However, imo it's a lot cleaner to have a specific type for the props and pass it, instead of creating it here only.
Furthermore, I also recommend getting in the habit of destructuring your props. It also allows for a cleaner implementation of all props.